Approximate_convex_decomposition/include/CGAL/approx_decomposition.h

```diff
@@ -0,0 +1,113 @@
+#ifndef CGAL_APPROXIMATE_CONVEX_DECOMPOSITION_APPROX_DECOMPOSITION_H
+#define CGAL_APPROXIMATE_CONVEX_DECOMPOSITION_APPROX_DECOMPOSITION_H
+
+#define CGAL_DEFAULT_CONCAVITY_THRESHOLD 0.5
```

Comment:
we try to avoid using macros when they are not needed

Reply:
Isn't it a bad practice to put the same constants everywhere? Maybe, something like constexpr double CGAL_DEFAULT_CONCAVITY_THRESHOLD = 0.5;  would be better?

Reply:
this value should not be spread all around, it should actually be at one place where the named parameter of the function is recovered so it is not needed to have it globally defined.

Reply:
![image](https://user-images.githubusercontent.com/12598936/40534520-f6d14f3a-600e-11e8-8c30-d4d5bdb52906.png)
Just look at those repeating numbers of number_of_clusters, smoothing_lambda, output_cluster_ids.
In case, when you want to change any of those, you have to change them at each function declaration.

Reply:
The discussion is meaningless provided no serious code doing something exists yet :)

Reply:
But IMO it's better to define some rules before writing any serious code :)